schedule/yast/lvm_raid1/lvm+raid1_opensuse.yaml

```diff
@@ -1,34 +1,33 @@
 ---
-name:           lvm+raid1@64bit
-description:    >
+  name : lvm +raid1 @64bit
+  description : >
   Validation of partitioning for raid1 on lvm
-  Installation of RAID1 using expert partitioner.
-vars:
-  RAIDLEVEL: 1
-  LVM: 1
-test_data:
-  !include: test_data/yast/lvm_raid1/lvm+raid1_opensuse.yaml
-schedule:
-  - installation/isosize
-  - installation/bootloader_start
-  - installation/welcome
-  - installation/online_repos
-  - installation/installation_mode
-  - installation/logpackages
-  - installation/system_role
-  - installation/partitioning
-  - installation/partitioning_raid
-  - installation/partitioning_finish
-  - installation/installer_timezone
-  - installation/user_settings
-  - installation/resolve_dependency_issues
-  - installation/installation_overview
-  - installation/disable_grub_timeout
-  - installation/start_install
-  - installation/await_install
-  - installation/logs_from_installation_system
-  - installation/reboot_after_installation
-  - installation/grub_test
-  - installation/first_boot
-  - installation/opensuse_welcome
-  - console/validate_lvm_raid1
+  Installation of RAID1 using expert partitioner .
+  vars :
+  RAIDLEVEL : 1
+  LVM       : 1
+  test_data :
+  !include : test_data / yast / lvm_raid1 / lvm +raid1_opensuse . yaml
+  schedule :
+  - installation / isosize
+  - installation / bootloader_start
+  - installation / welcome
+  - installation / online_repos
+  - installation / installation_mode
+  - installation / logpackages
+  - installation / system_role
+  - installation / partitioning
+  - installation / partitioning_raid
+  - installation / partitioning_finish
+  - installation / installer_timezone
+  - installation / user_settings
+  - installation / resolve_dependency_issues
+  - installation / installation_overview
+  - installation / disable_grub_timeout
+  - installation / start_install
+  - installation / await_install
+  - installation / logs_from_installation_system
+  - installation / reboot_after_installation
+  - installation / grub_test
+  - installation / first_boot
+  - console / validate_lvm_raid1
```

Comment:
Check this file, something strange happen with spaces.

Reply:
ee1b3981a resolved